sklearn/metrics/ranking.py

```diff
@@ -251,17 +255,21 @@ def roc_auc_score(y_true, y_score, average="macro", sample_weight=None):
     0.75
 
     """
-    def _binary_roc_auc_score(y_true, y_score, sample_weight=None):
+    def _binary_roc_auc_score(y_true, y_score, sample_weight=None,
+                              pos_label=None):
         if len(np.unique(y_true)) != 2:
             raise ValueError("Only one class present in y_true. ROC AUC score "
                              "is not defined in that case.")
 
         fpr, tpr, tresholds = roc_curve(y_true, y_score,
+                                        pos_label=pos_label,
                                         sample_weight=sample_weight)
         return auc(fpr, tpr, reorder=True)
 
+    _partial_binary_roc_auc_score = partial(_binary_roc_auc_score,
```

Comment:
we're using ``_average_binary_score`` here to do multi-label, right? In that case we don't need ``pos_label`` (it doesn't make sense). I would check type_of_target(y) here and if we are binary use ``pos_label`` and otherwise call ``_average_binary_score``.